runtime/src/lib.rs

```diff
@@ -754,14 +754,15 @@ parameter_types! {
     pub const FaucetPalletId: PalletId = PalletId(*b"ATFAUCET");
 }
 
-// impl pallet_faucet::Config for Runtime {
-//     type AccumulationPeriod = AccumulationPeriod;
-//     type RuntimeEvent = RuntimeEvent;
-//     type Currency = Balances;
-//     type PalletId = FaucetPalletId;
-//     type FaucetAmount = FaucetAmount;
-//     type WeightInfo = pallet_faucet::weights::FaucetWeight<Runtime>;
-// }
+#[cfg(any(feature = "testnet-runtime", feature = "devnet-runtime"))]
```

Comment:
I think it’s better to leave things as they are since we might not even need a faucet on the stagenet. This way, we clearly define which runtimes the faucet will be included in.

Reply:
Yes, we definitely don't want to keep faucet on stagenet.